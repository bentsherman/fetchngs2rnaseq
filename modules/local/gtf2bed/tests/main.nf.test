nextflow_process {

    name "Test Process GTF2BED"
    script "../main.nf"
    process "GTF2BED"

    test("sarscov2 - bam") {

        when {
            process {
                """
                input[0] = file(params.modules_testdata_base_path + "genomics/sarscov2/genome/genome.gtf", checkIfExists: true)
                """
            }
        }

        then {
            assertAll(
                { assert process.success },
                { assert snapshot(process.out).match() }
            )
        }

    }
}
