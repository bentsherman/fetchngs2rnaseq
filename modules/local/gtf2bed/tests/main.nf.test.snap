{
    "sarscov2 - bam": {
        "content": [
            {
                "0": [
                    "genome.bed:md5,d41d8cd98f00b204e9800998ecf8427e"
                ],
                "1": [
                    "versions.yml:md5,d04d815c224b980b6f3818d4b2fd4fe3"
                ],
                "bed": [
                    "genome.bed:md5,d41d8cd98f00b204e9800998ecf8427e"
                ],
                "versions": [
                    "versions.yml:md5,d04d815c224b980b6f3818d4b2fd4fe3"
                ]
            }
        ],
        "meta": {
            "nf-test": "0.8.4",
            "nextflow": "23.10.1"
        },
        "timestamp": "2024-03-07T15:00:44.458214"
    }
}